### PR TITLE
Update SV curation mechanism

### PR DESCRIPTION
**Pull request by alejoe91**
- get rid of legacy `JOT`
- use `GH` handle instead

### CODE REVIEW

**Comment by alejoe91** [comment]
@samuelgarcia @h-mayorquin 

this will fix the failing codecov tests: https://github.com/SpikeInterface/spikeinterface/actions/runs/4397330223/jobs/7700390707

**Comment by samuelgarcia** [comment]
Hi camarade.
If I understand we need an extra repo just have a hard saved json file for curation for testing ?
Why do'nt we included this little JSON file in the actual repo ? Will thsi file change often ?

**Comment by alejoe91** [comment]
> Hi camarade. If I understand we need an extra repo just have a hard saved json file for curation for testing ? Why do'nt we included this little JSON file in the actual repo ? Will thsi file change often ?

No it won't change! I can actually point to the same file! Thanks for the suggestion

File: spikeinterface/curation/tests/test_sortingview_curation.py [hunk 1]
```diff
@@ -44,44 +44,47 @@ def test_sortingview_curation():
...
-    assert len(sorting_curated_sha_accepted.unit_ids) == 3
-    assert len(sorting_curated_sha_mua.unit_ids) == 6
-    assert len(sorting_curated_sha_mua1.unit_ids) == 5
+    assert len(sorting_curated_json_accepted.unit_ids) == 3
+    assert len(sorting_curated_json_mua.unit_ids) == 6
+    assert len(sorting_curated_json_mua1.unit_ids) == 5
+
+    # from GH
+    # curated link: https://figurl.org/f?v=gs://figurl/spikesortingview-10&d=sha1://bd53f6b707f8121cadc901562a89b67aec81cc81&label=SpikeInterface%20-%20Sorting%20Summary&s={%22sortingCuration%22:%22gh://SpikeInterface/test-sv-sorting-curation/main/test/curation-test.json%22}
+    gh_uri = "gh://SpikeInterface/spikeinterface/master/spikeinterface/curation/tests/sv-curation-test.json"
```
Comment (h-mayorquin) [inline spikeinterface/curation/tests/test_sortingview_curation.py:71]: Isn't this pointing this to something that does not exist yet? (this is not yet in master). Is that the reason why the testing time decreased so drastically? 

Should we still keep the commented code above? If so, could you write a longer comment explaining what it is to future onlookers.

**Review by h-mayorquin — APPROVED** [review]
This seems OK to me.

**Pull request merged**